azure/src/index.ts

```diff
@@ -4,6 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 export * from './createAzureClient';
+export { registerAzureUtilsExtensionVariables } from './extensionVariables';
```

Comment:
nit: Undo this change since it's just reordering the imports?

Reply:
I mean, it was in the wrong spot before anyway. It's in alphabetical order now.

Reply:
Ok fine. I'm just being nit-picky about this PR touching all these extra files. Kinda messes with the git blame. I'd prefer if we fixed all the imports in a separate PR but it's fine as is.